{{cookiecutter.project_slug}}/fabfile.py

```diff
@@ -242,3 +242,22 @@ def get_media(directory=''):
             media_bucket=env.media_bucket, media=env.media, directory=directory
         )
     )
+
+
+@task
+@runs_once
+@roles('web')
+def get_env():
+    """ Get the current environment variables, ready for local export."""
+    env.output_prefix = False
+    run('export | sed -e "s/declare -x/export/g"')
+    
+
+@task
+@runs_once
```

Comment:
Just realised, one final minor thing!

Remove runs_once and roles decorators - it's a local task anyway.